config/manager/manager_pull_policy.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
       - name: manager
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Added by mistake?

Reply:
need to sort out that `make test-e2e` so it doesn't do this.

Reply:
excised